build/azure-pipelines/win32/sql-product-build-win32.yml

```diff
@@ -228,13 +228,46 @@ steps:
       MaxRetryAttempts: 20
     condition: and(succeeded(), eq(variables['signed'], true))
 
+  - task: UseDotNet@2
```

Comment:
@cheenamalhotra Out of curiosity - did you verify if we actually needed this? Or are you just explicitly adding it to avoid future breaks if/when they remove .NET 6 from the build machines?

Reply:
We're only setting ESRPPKI and related env variables to be consumed by inno_updater.exe when packaging. So the ESRPCLI installed on system can be invoked by inno_updater.exe during packaging. I do think we should keep it installed, as in future if we need to hotfix any releases, we don't want pipelines to fail due to images being updated behind the scenes and YMLs not considerate of that.

I rather think YML should take care of this more strongly as image update is out of our control.